### PR TITLE
Support for the new longer ecs arn format

### PR DESCRIPTION
**Pull request by jonathonsim**
The format of ECS ARN's is changing.  
https://aws.amazon.com/ecs/faqs/#Transition_to_new_ARN_and_ID_format

The TLDR is that where the old format of the ARN of a container instance looked like
           ` arn:aws:ecs:us-east-1:111111111111:container-instance/00c4a1c9-0c10-498b-b8c8-d5dc44c61ee0`

the new format contains the cluster name, like this:
            `arn:aws:ecs:us-east-1:111111111111:container-instance/ClusterName/00c4a1c9-0c10-498b-b8c8-d5dc44c61ee0`

It's opt-in for existing accounts but if you do opt-in the draining lambda will no longer be able to stop tasks you've started manually on the instance as it ends out picking up the Cluster name rather than the instance id when looking for it in the new format ARNs

### CODE REVIEW

**Comment by endofcake** [comment]
LGTM 👍

**Comment by timonwalkley** [comment]
Hi @jonathonsim 👋, thanks for the PR!
